### PR TITLE
add support for nash market orders

### PR DESCRIPTION
**Pull request by jankjr**

### CODE REVIEW

**Comment by jankjr** [comment]
This depends on an update to the nash-protocol

File: src/nash/mod.rs [hunk 1]
```diff
@@ -261,25 +261,37 @@ impl ExchangeAccount for Nash {
...
-        Err(OpenLimitError::MissingImplementation(err))
+    async fn market_buy(&self, req: &OpenMarketOrderRequest) -> Result<Order> {
+        let req: nash_protocol::protocol::place_order::MarketOrderRequest =
+            Nash::convert_market_request(req, nash_protocol::types::BuyOrSell::Buy);
+
+        let resp = self.transport.run(req).await;
+        Ok(
+            Nash::unwrap_response::<nash_protocol::protocol::place_order::PlaceOrderResponse>(
+                resp,
+            )?
```
Comment (notdanilo) [inline src/nash/mod.rs:292]: Can you directly use `val` instead of `OK(val)?`? It looks redundant or I am missing the point.

**Review by notdanilo — APPROVED** [review]
It looks good! I just left a minor consideration about the use of `OK()?`

**Comment by Ejhfast** [comment]
This looks good, just make sure to change Cargo.toml back before merge

**Pull request merged**